### ISSUE TITLE
WebIDL rendering adds "<span class='idlSectionComment'>" to output

### ISSUE THREAD

**Issue by mgiuca** [w3c/respec#1717]
## Important info

* URL to affected spec: https://wicg.github.io/web-share-target/
* ReSpec version: 22.0.0
* [x] I did a "hard refresh", but it's still busted. 

## Description of problem

What happened (e.g., it crashed)?: 
WebIDL with a "partial dictionary" after another dictionary has "`<span class='idlSectionComment'>`
`</span>`" in between the two dictionaries.

But what I was expecting?:
Not the span thing.

Optional, steps to reproduce:

```html
    <pre class="idl">
      dictionary OneThing {
        int x;
      };
      partial dictionary AnotherThing {
        int y;
      };
    </pre>
```

Renders as:

![image](https://user-images.githubusercontent.com/228433/41958595-db829390-7a2d-11e8-86cb-663654a7bdb4.png)

**Comment by marcoscaceres** [comment]
Small regression, @saschanaz

**Comment by marcoscaceres** [comment]
Will release in about 10 hours

**Comment by marcoscaceres** [comment]
(🤞) fixed in 22.0.1.
